### PR TITLE
spanishdict.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/129860

<details><summary>Screenshot:</summary>

Screenshot 2022-09-16 at 17 06 33

</details><br/>

### CODE REVIEW

**Comment by Alex-302** [comment]
Cannot reproduce this popup. Is it really annoying?

File: SpywareFilter/sections/cookies.txt [hunk 1]
```diff
@@ -170,6 +170,7 @@ $cookie=ev_sid,domain=ideal.es|elcorreo.com
 ||tf1.fr^$cookie=ABTastySession
 ||tf1.fr^$cookie=ABTasty
 ||tf1.fr^$cookie=gguserid
+||ingles.com^$cookie=logglytrackingsession
```
Comment (Alex-302) [inline SpywareFilter/sections/cookies.txt:173]: Please move aboe with a comment(added a comment to this group few moments ago)

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/cookies.txt [hunk 2]
```diff
@@ -10,6 +10,9 @@
 !------------------------------
 ! NOTE: Cookies specific
 !
+! ingles.com/spanishdict.com - tracking
+||ingles.com^$cookie=logglytrackingsession
+||spanishdict.com^$cookie=logglytrackingsession
```
Comment (Alex-302) [inline SpywareFilter/sections/cookies.txt:15]: Can be merged into one rule
```suggestion
$cookie=logglytrackingsession,domain=ingles.com|spanishdict.com
```
or can be changed to generic.
Description:
https://www.plotto.com/cookies

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/cookies.txt [hunk 3]
```diff
@@ -199,6 +198,10 @@ $cookie=ev_sid,domain=ideal.es|elcorreo.com
 !------------------------------
 ! NOTE: Cookies generic rules
 !
+! plotto.com
+! https://www.plotto.com/cookies
```
Comment (Alex-302) [inline SpywareFilter/sections/cookies.txt:202]: This is just a link to description from random site.

File: BaseFilter/sections/css_extended.txt [hunk 4]
```diff
@@ -6,6 +6,8 @@
 ! Bad: example.org##main > div.ad (should be in specific.txt)
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/129860
+ingles.com,spanishdict.com#?#body > div.ReactModalPortal:has(> .ReactModal__Overlay > .ReactModal__Content > div[class] > div[class] > div[class] > div[class] > iframe)
```
Comment (zloyden) [inline BaseFilter/sections/css_extended.txt:10]: I'd specify this `iframe` as the rule below
```
! https://github.com/AdguardTeam/AdguardFilters/issues/129860
ingles.com,spanishdict.com#?#body > div.ReactModalPortal:has(> div.ReactModal__Overlay > div.ReactModal__Content > div[class] iframe[title^="video"][srcdoc*="birthdayDog.mp4"])
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/css_extended.txt (hunk 4 above)
Comment (jellizaveta) [reply BaseFilter/sections/css_extended.txt:10]: There's more than one type of iframe

**Review by Alex-302 — APPROVED** [review]

**Comment by sugarfreesweetener** [comment]
@Yuki2718

**Pull request merged**